documentation-site/components/overrides.js

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import React from 'react';
+import {Block} from 'baseui/block';
+import {StatefulPopover, TRIGGER_TYPE, PLACEMENT} from 'baseui/popover';
+import {Card, StyledBody} from 'baseui/card';
+
+function isStyledExport(exportName) {
+  return exportName.startsWith('Styled');
+}
+
+function getOverrideName(exportName) {
+  return exportName.replace('Styled', '');
```

Comment:
This may not be true for all components.

Reply:
I've just gone through all the components, and it seems only the Block component does not export styled components, so I think we should be good with this one